client/rpc/src/lib.rs

```diff
@@ -71,6 +71,13 @@ pub enum SaveFileToDisk {
     IncompleteFile(IncompleteFileStatus),
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub enum GetFileFromFileStorageResult {
```

Comment:
Given that this struct is the exact same as the one above, how about changing the name and reusing it?

Reply:
With the latest changes it's more differentiated so I think it makes sense to keep both